web/src/documents.ts

```diff
@@ -64,6 +64,32 @@ export interface ContentChangeEvent extends CustomEvent {
   detail: ViewUpdate | string
 }
 
+/**
+ * Create a new document
+ *
+ * Optionally pass the content for the new document.
+ * If `content` is not a string and format is 'json' of undefined,
+ * then `content` will be stringify to JSON.
```

Comment:
@alex-ketch Note that I have changed the parameter of this so that the caller does not need to do `JSON.stringify` first.